Makefile

```diff
@@ -0,0 +1,21 @@
+
+# Image URL to use all building/pushing image targets
+IMG ?= raushan2016/etcdexplore
```

Comment:
Pls change it to `registry.cn-hangzhou.aliyuncs.com/ringtail/lucas`

Reply:
done